### ISSUE TITLE
Goalkeeper enhancement: high dive and low dive

### ISSUE THREAD

**Issue by PeyoteJr** [romdi/IOS#526]
More often than not, as a goalkeeper you'll be tempted to get high balls by pressing w+jump: in the current version of the game this leads to the gk diving on the ground.

My suggestion is to split the dives into high dives (currently every dive except the forward one) and low dives. 

I suggest to make it so that:

-  direction + jump(space) = high dive 
- direction + slide(alt) = low dive

This way goal keepers could retain more control on their action and maybe try and anticipate crosses.
Also, making low side dives can help in dealing with dribblers in the box, but could maybe cause a penalty if you miss.

Keep up the good work!

**Comment by Aryan14** [comment]
On 13 September, 2017 I tried to reach you in #feedback (Discord) the following:

> @PeyoteJr Saw your ticket week ago. Not sure what you exactly mean. You can already revert the keeping keys to the old way. (W and S for jumping instead of forward/backward dive). On the other hand you seem to suggest a high/low dive system for keepers. The idea is good but we had this in the past but felt too difficult for keepers.

Still waiting for your response.

**Comment by PeyoteJr** [comment]
It was indeed a noob complaining on something that he should've played more to understand it fully: 
TLDR: Switched to the "old" config has solved all my problems. Regarding the low/high dives, it is still a valid idea from my POV, but I can't suggest a way on how to implement it properly other than the one in my former comment, which I find, now that I have more experience with the game, to be a bit clunky.
Right now I can't think of another way of doing this kind of enhancement, but if I manage to come up with something I'll surely let you know.
